ydb/tests/functional/tenants/test_db_counters.py

```diff
@@ -129,3 +129,239 @@ def test_case(self):
             'Responses/Bytes',
         }
         self.check_db_counters(sensors_to_check, 'kqp')
+
+
+class StorageCounters(BaseDbCounters):
+    def setup_method(self, method=None):
+        self.database = f"/Root/users/{self.__class__.__name__}_{method.__name__}"
+        logger.debug(f"Create database {self.database}")
+        self.cluster.create_database(self.database, storage_pool_units_count={"hdd": 1, "hdd1": 1})
+
+        self.cluster.register_and_start_slots(self.database, count=1)
+        self.cluster.wait_tenant_up(self.database)
+
+    def teardown_method(self, method=None):
+        logger.debug(f"Remove database {self.database}")
+        self.cluster.remove_database(self.database)
+        self.database = None
```

Comment:
Consider switching to using fixtures from `ydb/tests/library/harness/ydb_fixtures.py`. And switching to standalone test functions.